base_station/gui/README.md

```diff
@@ -3,13 +3,34 @@
 ### Base Station GUI LCMs
 
 #### All GUIs:
-Camera 
+ArmControlState
+"/arm_control_state"
+File: ArmControlState.lcm
+Subscriptions: N/A
+Publishers: ArmControls.vue
+Usage: Selecting open/closed loop
+
+ArmControlState
+"/arm_control_state_to_gui"
+File: ArmControlState.lcm
+Subscriptions: ArmControls.vue, SAArm.vue,
+Publishers: N/A
+Usage: Determining open/closed loop
+
+Camera (this is no longer used, but subscribed)
 "/camera_servos"
 File: CameraServos.lcm
 Subscritions: SATask.vue, AutonTask.vue, RATask.vue
 Publishers: N/A
 Usage: 
 
+Cameras
+"/cameras_cmd"
+File: Cameras.lcm
+Subscriptions: N/A
```

Comment:
jetson/cameras? Ask Guthrie which program listens to this

Reply:
yes jetson/cameras